### DISCUSSION TITLE
How to add video in Notes?

### DISCUSSION THREAD

**Discussion by MaheshManoharan** [singerdmx/flutter-quill#349] [Mobile]
Maybe I am wrong, but I only get an alert dialog with "paste a link" and it's working with youtube videos. But what to do for adding a local video?

**Comment by singerdmx** [comment]
Please try running `example` directory successfully before creating an topic here

**Reply by MaheshManoharan** [reply]
I already did bro. only url videos are playing while mobile data is on. But I am asking, If there is any way to add video in our mobile devices into notes. If I am wrong, I am sorry.

**Reply by singerdmx** [reply]
It already works that way

**Reply by MaheshManoharan** [reply]
Thanks bro. That was my problem.
